lib/Sema/Constraint.cpp

```diff
@@ -322,11 +322,12 @@ Constraint *Constraint::clone(ConstraintSystem &cs) const {
   llvm_unreachable("Unhandled ConstraintKind in switch.");
 }
 
-void Constraint::print(llvm::raw_ostream &Out, SourceManager *sm) const {
+void Constraint::print(llvm::raw_ostream &Out, SourceManager *sm, bool repeatedLocator) const {
   // Print all type variables as $T0 instead of _ here.
   PrintOptions PO;
   PO.PrintTypesForDebugging = true;
 
+  ConstraintLocator *firstLocator;
```

Comment:
Since `firstLocator` applies only no conjunctions and disjunctions let's declare it inside of `if` statement.